terraform/iam_templates/nextstrain_jobs_access_to_logs.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "logs:GetLogEvents",
+                "logs:FilterLogEvents",
+                "logs:DeleteLogStream"
+            ],
+            "Resource": [
+                "arn:aws:logs:*:*:log-group:/aws/batch/job",
+                "arn:aws:logs:*:*:log-group:/aws/batch/job:log-stream:*"
```

Comment:
are these log names prefixed with the job queue or anything?  can we limit the scope?

Reply:
these were the policies that were in the [nextstrain docs](https://docs.nextstrain.org/projects/cli/en/latest/aws-batch/#nextstrainjobsaccesstologs), I didn't modify them when implementing them here. I can play around with limiting the scope more, the log group _should_ just be `/aws/batch/job`

Reply:
the log stream name is always `nextstrain-job-definition/default/*`, can we take advantage of that somehow?